package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "universal-navigation",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/topcoder-platform/universal-navigation.git",
+  "license": "MIT"
```

Comment:
Just curious if the MIT license is right for company IP. I don't know what the other options are but figured I'd ask.

Reply:
good question. i have no idea--just left it as the default.